README.md

```diff
@@ -1,2 +1,252 @@
+    ·
+    <a href="https://github.com/gafda/example-repo/issues/new?labels=enhancement&template=feature-request.md">Request Feature</a>
+  </p>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgments">Acknowledgments</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+[![Product Name Screen Shot, may be ani-gif][product-screenshot]](https://example.com)
+
+This project is designed to provide a comprehensive solution for [briefly describe the purpose of the project]. It aims to [mention the main goals or objectives of the project].
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Built With
+
+This section should list any major frameworks/libraries used to bootstrap your project. Leave any add-ons/plugins for the acknowledgements section. Here are a few examples.
+
+* [![dotNetCore][dotnetcore-shield]][dotnetcore-url]
+* [![Bootstrap][bootstrap-shield]][bootstrap-url]
+* [![Docker][docker-shield]][docker-url]
+* [![Kubernetes][kubernetes-shield]][kubernetes-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Hardware:
+* 40 TOP NPU
+* nVidia RTX20+
+* 4GB VRAM+
+* CPU Cores 4x+
+* ...
+
+Software:
+* DotNet Core 6.0
+* DirectX v12.1
+* TensorFlow 2.0
+* ...
+
+Extra:
+* Internet access
+* ...
+
+### Installation
+
+_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
+
+1. Get a free API Key at [https://example.com](https://example.com)
+2. Clone the repo
+    ```sh
+    git clone https://github.com/github_username/repo_name.git
+    cd repo_name
+    ```
+3. Install libraries
+    ```sh
+    dotnet restore
+    ```
+4. Enter your API in `app.settings`
+    ```json
+    API_KEY: "ENTER YOUR API"
+    ```
+5. Build the project
+    ```sh
+    dotnet build
+    ```
+6. Run the project
+    ```sh
+    dotnet run
+    ```
+7. Open your browser and navigate to `http://localhost:5000`
+8. Enjoy!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
## Structure of the project

How the project is structured.

Reply:
Do you mean this:
https://github.com/gafda/example-repo/blob/task-readme-and-extras/source/HLD.md